ampd-handlers/src/bin/evm/main.rs

```diff
@@ -0,0 +1,103 @@
+mod error;
+mod handler;
+
+use std::time::Duration;
+
+use ampd::evm::finalizer::Finalization;
+use ampd::json_rpc;
+use ampd::url::Url;
+use ampd_sdk::config;
+use ampd_sdk::runtime::HandlerRuntime;
+use axelar_wasm_std::chain::ChainName;
+#[cfg(debug_assertions)]
+use dotenv::dotenv;
+use error_stack::{Result, ResultExt};
+use ethers_providers::Http;
+use serde::{Deserialize, Serialize};
+use tokio_util::sync::CancellationToken;
+use tracing::Level;
+
+use crate::error::Error;
+use crate::handler::Handler;
+
+#[derive(Debug, Deserialize, Serialize)]
+struct EvmHandlerConfig {
+    #[serde(deserialize_with = "Url::deserialize_sensitive")]
+    rpc_url: Url,
+    #[serde(with = "humantime_serde")]
+    #[serde(default = "default_rpc_timeout")]
+    rpc_timeout: Duration,
+    #[serde(default)]
+    finalization: Finalization,
+}
+
+fn default_rpc_timeout() -> Duration {
+    Duration::from_secs(3)
+}
+
+fn build_handler(
+    runtime: &HandlerRuntime,
+    chain_name: ChainName,
+    config: EvmHandlerConfig,
+) -> Result<Handler<json_rpc::Client<Http>>, Error> {
+    let rpc_client = json_rpc::Client::new_http(
+        config.rpc_url,
+        reqwest::ClientBuilder::new()
+            .connect_timeout(config.rpc_timeout)
+            .timeout(config.rpc_timeout)
+            .build()
+            .change_context(Error::HandlerStart)?,
+        runtime.monitoring_client.clone(),
+        chain_name.clone(),
+    );
+
+    let handler = Handler::builder()
+        .verifier(runtime.verifier.clone())
+        .voting_verifier_contract(runtime.contracts.voting_verifier.clone())
+        .chain(chain_name)
+        .finalizer_type(config.finalization)
+        .rpc_client(rpc_client)
+        .monitoring_client(runtime.monitoring_client.clone())
+        .build();
+
+    Ok(handler)
+}
+
+fn init_tracing(max_level: Level) {
+    tracing::subscriber::set_global_default(
+        tracing_subscriber::FmtSubscriber::builder()
+            .with_max_level(max_level)
+            .finish(),
+    )
```

Comment:
datadog won't be able to process this properly (needs json). Can you add the json/text config we also in base ampd?